docs/docs-content/clusters/edge/local-ui/cluster-management/link-hosts.md

```diff
@@ -0,0 +1,157 @@
+
+Linked hosts will sync images for the Palette agent, provider images, cluster definitions, as well as host status with
+each other. In a group of linked hosts that have not formed a cluster, only the leader node has access to functionality
+such as change host settings, upload content, and create clusters. Once a cluster is created, only the control plane
+nodes have access to features such as cluster management and can change host settings.
+
+## Link Hosts
+
+### Limitations
+
+- All hosts must be deployed in the same deployment mode. For more information, refer to
+  [Deployment Modes](../../../../deployment-modes/deployment-modes.md).
+
+- For hosts that are deployed in agent mode, all hosts must share the same Operating System (OS).
+
+### Prerequisites
+
+- Two or more hosts deployed in the same deployment mode on the same network. For more information, refer to
+  [Appliance Mode Installation](../../site-deployment/stage.md) or
+  [Agent Mode Installation](../../../../deployment-modes/agent-mode/install-agent-host.md).
+
+- All hosts must be idle. They cannot have any current cluster workloads, or are linked to another node. Refer to
+  [Delete a Cluster](./delete-cluster.md) and [Unlink Hosts](#unlink-hosts) to learn how to delete a cluster and unlink
+  a host to free it up for linking.
+
+### Procedure
+
+1. Decide on a node that you plan to use as the leader of the group. Log in to
+   [Local UI](../host-management/access-console.md) of that node. The leader node may or may not already be in a
+   cluster, but the follower nodes must be idle and cannot be in a cluster.
+
+2. From the left **Main Menu**, click **Linked Edge Hosts**.
+
+3. Click **Generate token**. This will generate a token you will use to link this host with other hosts. The token
+   contains the encrypted IP address of the node, as well as an OTP that will expire in two minutes.
+
+4. Click the **Copy** button to copy the token.
+
+5. Log in to [Local UI](../host-management/access-console.md) on the node that you want to link to the leader node.
+
+6. From the left **Main Menu**, click **Linked Edge Hosts**.
+
+7. Click **Link this device to another**.
+
+8. In the pop-up box that appears, enter the token you coped from the leader node.
+
+9. Click confirm.
+
+10. Repeat this process for every node you want to link to the leader node.
+
+### Validate
+
+1. Log in to [Local UI](../host-management/access-console.md) on the leader node.
+
+2. From the left **Main Menu**, click **Linked Edge Hosts**.
+
+3. Confirm that all nodes you linked together show up in the **Linked Edge Hosts** table.
+
+## Unlink Hosts
+
+You can unlink a host to either link it to another host or to use it for independent workloads. You can only unlink a
+follower host. Leader hosts cannot be unlinked.
+
+You can unlink a follower host on the follower host itself or from the leader node.
+
```

Comment:
- If follower is "in-use" it cannot be unlinked(disabled in UI). On scale-down of this host the "unlink" option will be enabled in UI.
- If leader is "in-use" and follower is "ready" the follower can be unlinked from leader or follower